formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/services/DynamoUpdateService.scala

```diff
@@ -138,9 +138,16 @@ case class DynamoUpdateService(
       case Left(err) => Left(err)
       case Right(response) =>
         val forms = response.forms
-        val formResults = forms.map { form =>
-          logger.info(s"Processing results for form ${form.id}")
-          writeSubmissions(form, lastUpdate, token = token)
+        val formResults: Seq[Either[Throwable, Unit]] = forms.map { form =>
+          // The amount of new submissions to these forms often exceeds an apparent formstack limit on how many can be fetched in a single query (even if the api call is paginated)
+          // The submissions would be ignored anyway since the form doesn't have any relevant data, so we just skip them at the form level to avoid the problem for now.
+           if (form.name.toLowerCase.startsWith("ybtj")) {
```

Comment:
Perhaps extract this condition into its own method called something like `mightHaveEmailField(form)` and swap the outcomes.